src/constants/routes.ts

```diff
@@ -1,3 +1,5 @@
-export enum Routes {
+export enum AppRoutes {
+  Editor = 'n/:id',
+  Root = '/',
   ShareFromList = 'share'
```

Comment:
shareFromList should be `/share?n=:id`, and the route would get the data by itself, as it should in idiomatic routing